lambdas/service/app.py

```diff
@@ -1155,6 +1185,7 @@ def add_all_results_to_cart(cart_id):
             "statusUrl": "https://status.url/resources/carts/status/{token}"
         }
     """
+    cart_id = None if cart_id == 'default' else cart_id
```

Comment:
This change is also not in the discussion but since we are doing this in the method that adding a single item, I think we should do the same for the batch operation.